### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by Luchy-B**
- Add Styling

### CODE REVIEW

File: src/Components/Boolist/Book.js [hunk 1]
```diff
@@ -1,28 +1,60 @@
...
+      </div>
+      <div className="eachBookHr" />
+
+      <div className="updateContainer">
+        <div>
+          <h5>CURRENT CHAPTER</h5>
+          <p>{`Chapter ${randomCap}`}</p>
+          <button type="button" className="btn">UPDATE PROGRESS</button>
+        </div>
+      </div>
```
Comment (danmainah) [inline src/Components/Boolist/Book.js]: - [ ] @Luchy-B kindly ensure your design matches the design given, your font, text color, and progress bar circle color, and also  update the progress button color, also check the background color highlighted in both screenshots check the comparison below for a better understanding:




Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

File: src/Components/Boolist/NewBookForm.js [hunk 2]
```diff
@@ -25,15 +26,31 @@ const NewBookForm = () => {
...
+          <option value="fiction">Fiction</option>
+          <option value="non-fiction">Non-Fiction</option>
+          <option value="mystery">Mystery</option>
+          <option value="sci-fi">Romance</option>
+          <option value="sci-fi">Fantasy</option>
+        </select>
+        <br />
+        <button type="submit" onClick={handleClick} className="btn">ADD BOOK</button>
       </div>
     </form>
```
Comment (danmainah) [inline src/Components/Boolist/NewBookForm.js]: - [ ] @Luchy-B great job on your design however  the form occupies larger width than in the design also the button is small kindly increase its size and ensure it's color matches the design given,


Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Luchy-B , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] @Luchy-B kindly ensure there are no errors when running your app, I encountered the error below when running your app 
image


- [ ] @Luchy-B great work on your design however, your navbar design differs from the design given, the color of navbar items, border, and user budget on the right and when selecting a nav item it's not highlighted(made brighter than the other item, eg in the design below books is darker than categories), kindly check the comparison below

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by IngDavidTM — APPROVED** [review]
Hi @Luchy-B  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**